src/main/java/org/datadog/jenkins/plugins/datadog/traces/DatadogTraceBuildLogic.java

```diff
@@ -4,6 +4,8 @@
 import static org.datadog.jenkins.plugins.datadog.traces.GitInfoUtils.normalizeBranch;
 import static org.datadog.jenkins.plugins.datadog.traces.GitInfoUtils.normalizeTag;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
```

Comment:
Does this package need to be added to `pom.xml`?

Reply:
Not needed because is inherited by the jenkins dependencies